sqlite3_opt_serialize.go

```diff
@@ -0,0 +1,68 @@
+// +build !sqlite_omit_deserialize
```

Comment:
This is not quite right. This file should be used if `!libsqlite3` or if `sqlite_serialize` (or both). That way it is opt-in for anyone linking with their system version, and always included otherwise. https://github.com/mattn/go-sqlite3/pull/1089#issuecomment-1238808014

Reply:
>This is not quite right. This file should be used if !libsqlite3 or if sqlite_serialize (or both). 

I presume you mean `sqlite_omit_deserialize` since I was asked to add it in https://github.com/mattn/go-sqlite3/pull/1089#issuecomment-1238743103

Reply:
See the subsequent comment from @mattn. https://github.com/mattn/go-sqlite3/pull/1089#issuecomment-1238808014

Reply:
Sorry, I'm not following. Can you be more explicit? What build tags do you want added, exactly?

Reply:
If they aren't using libsqlite3, then these functions are always included (there is no opt-out).
If they are using libsqlite3, then they must opt-in via a build tag (`sqlite_serialize`) to preserve backwards compatibility in all cases.

Thus this condition should be `// +build !libsqlite3 sqlite_serialize`.